iaso/cli/dump2datamine.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import click
+
+from ..click.lazy import lazy_import
+from ..click.docker import wrap_docker, DockerPathExists
+
+lazy_import(
```

Comment:
Let's have a chat on these lazy import mechanism in our next office day

Reply:
The lazy import mechanism was a necessary approach to significantly improve the load times of the command line tool. Without it, Python would import all dependencies recursively initially, taking up a few seconds. Especially for supporting shell completion, this is unacceptable. There is a similar mechanism in importlib (see https://docs.python.org/3/library/importlib.html#importlib.util.LazyLoader and https://stackoverflow.com/a/51126745) which work in a similar way. Essentially, the goal is to postpone as much of the module loading as possible.

Reply:
Moreover, I am using a lot of `from mod import obj` syntax, which is not supported by the `LazyLoader`.